.env.template

```diff
@@ -4,3 +4,8 @@ FLASK_ENV=development
 
 # Change the following values for local development.
 SECRET_KEY=secret-key
+TRELLO_TODO='610beb5a99354405b1f4ec6f'
+TRELLO_DOING='610beb5a99354405b1f4ec70'
+TRELLO_DONE='610beb5a99354405b1f4ec71'
+TRELLO_KEY=
+TRELLO_TOKEN=
```

Comment:
It looks like you also have the `TRELLO_BOARD` and `TRELLO_ID` keys that needs to be set